core/src/main/java/org/mskcc/cbio/portal/dao/DaoCnaEvent.java

```diff
@@ -64,8 +64,13 @@ public static void addCaseCnaEvent(CnaEvent cnaEvent, boolean newCnaEvent) throw
                     Integer.toString(cnaEvent.getCnaProfileId())
                     );
 
-            if ((cnaEvent.getDriverFilter() != null && !cnaEvent.getDriverFilter().isEmpty())
-                || (cnaEvent.getDriverTiersFilter() != null && !cnaEvent.getDriverTiersFilter().isEmpty())) {
+            if ((cnaEvent.getDriverFilter() != null
+                && !cnaEvent.getDriverFilter().isEmpty()
+                && !cnaEvent.getDriverFilter().toLowerCase().equals("na"))
+                || 
+                (cnaEvent.getDriverTiersFilter() != null
+                && !cnaEvent.getDriverTiersFilter().isEmpty()
+                && !cnaEvent.getDriverTiersFilter().toLowerCase().equals("na"))) {
```

Comment:
I believe there might be some class with different potential N/A values that one can use here. Not entirely sure tho-  can look more later

Reply:
can't find any util class that provides this NA checking functionality:

https://github.com/cBioPortal/cbioportal/search?p=5&q=nan

So I think this is ok. Just wondering if it can't also be `N/A` or `NaN`?

Reply:
Well, in FileFormats.md these fields can be '' or 'NA'. This is enforced by the validator. This is the relevant code:

line 1220
```
NULL_DRIVER_TIERS_VALUES = ('', 'NA')
```

line 1326
```
        if value not in self.NULL_DRIVER_TIERS_VALUES:
            self.logger.info('Values contained in the column cbp_driver_tiers that will appear in the "Mutation Color" '
                             'menu of the Oncoprint',
                             extra={'line_number': self.line_number, 'column_number': self.cols.index('cbp_driver_tiers'), 'cause': value})
            self.tiers.add(value)
```

Reply:
Thanks!